### DISCUSSION TITLE
GPG Signed commits using Official bot

### DISCUSSION THREAD

**Discussion by ps-occrp** [renovatebot/renovate#25639] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is it possible to have official renovate bot (Mend Github App) sign commits when creating PRs?

### What you tried so far.

Checked documentation and searched in github

**Comment by rarkins** [comment]
Bot commits are already verified, eg https://github.com/renovatebot/renovate/pull/25633/commits
